packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/tasks/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,11 @@
+{
+  "pci_databases_connectors_tasks_back": "Connectors",
+  "pci_databases_connectors_tasks_id": "Id",
+  "pci_databases_connectors_tasks_trace": "Trace",
+  "pci_databases_connectors_tasks_status": "Statut",
+  "pci_databases_connectors_tasks_restart": "Redémarrer",
+  "pci_databases_connectors_tasks_modify_configuration": "Modifier le connector",
+  "pci_databases_connectors_tasks_title": "Tasks du connector \"{{name}}\"",
+  "pci_databases_connectors_tasks_id_number": "Task n°{{id}}",
```

Comment:
Question: Is it required to translate this text? if not can you move to constant file.

Reply:
I think that we need translation here, yes, as I'm not sure that "n°" works for every language.